x/stakeibc/keeper/icacallbacks_claim.go

```diff
@@ -56,7 +55,7 @@ func ClaimCallback(k Keeper, ctx sdk.Context, packet channeltypes.Packet, ack *c
 	txMsgData, err := icacallbacks.GetTxMsgData(ctx, *ack, k.Logger(ctx))
 	if err != nil {
 		k.Logger(ctx).Error(fmt.Sprintf("failed to unmarshal txMsgData, packet %v", packet))
-		return sdkerrors.Wrap(icacallbackstypes.ErrTxMsgData, err.Error())
+		return fmt.Errorf(icacallbackstypes.ErrTxMsgData.Error(), err.Error())
```

Comment:
@hoangdv2429: It's should be `return fmt.Errorf("%s: %s", err.Error(), icacallbackstypes.ErrTxMsgData.Error())`